### PR TITLE
Cyril HW#7 (Redis)

### PR DESCRIPTION
**Pull request by CyrilKuzmin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+secret/.token
+secret/.db_conn
```
Comment (subtle-byte) [inline .gitignore:2]: A little improvement: just `secret` also works

File: internal/repository/redis_cache.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		return res, false
+	}
+	if err != nil {
+		c.logger.Error().Err(err).Msg("error during the getting price from cache")
+		return res, false
+	}
+	json.Unmarshal([]byte(val), &res)
+	if err != nil {
+		c.logger.Error().Err(err).Msg("cannot unmarsal redis data")
+	}
```
Comment (subtle-byte) [inline internal/repository/redis_cache.go]: Seems like `err =` is forgot

**Comment by subtle-byte** [comment]
Redis works, good code!

File: internal/repository/redis_cache.go (hunk 2 above)
Comment (CyrilKuzmin) [reply internal/repository/redis_cache.go]: Fixed. Thanks :)

File: .gitignore (hunk 1 above)
Comment (CyrilKuzmin) [reply .gitignore:2]: I often switch between different laptops and/or its OSs. And .gitignore works for me as a note about missing files :)

**Pull request closed without merging**